utils/saveData.js

```diff
@@ -0,0 +1,17 @@
+const windowGlobal = typeof window !== 'undefined' && window;
+
+export const saveData = (formPath, formData, step) => {
+  const data = {
+    step: step,
+    formData: formData,
+  };
+  windowGlobal.localStorage.setItem(formPath, JSON.stringify(data));
+};
+
+export const getData = (path) => {
+  return JSON.parse(windowGlobal.localStorage.getItem(path));
```

Comment:
localStorage iterations should be around `try/catch` as it could be that the data is corrupted or the user has the localstorage disabled.